Python/Tests/Analysis/AstAnalysisTests.cs

```diff
@@ -399,37 +445,43 @@ public class AstAnalysisTests {
         private void AstBuiltinScrape(PythonVersion version) {
             version.AssertInstalled();
             using (var analysis = CreateAnalysis(version)) {
-                var fact = (AstPythonInterpreterFactory)analysis.Analyzer.InterpreterFactory;
-                var interp = (AstPythonInterpreter)analysis.Analyzer.Interpreter;
-
-                var mod = interp.ImportModule(interp.BuiltinModuleName);
-                var modPath = fact.GetCacheFilePath(fact.Configuration.InterpreterPath);
-                if (File.Exists(modPath)) {
-                    Console.WriteLine(File.ReadAllText(modPath));
-                }
-                Assert.IsInstanceOfType(mod, typeof(AstBuiltinsPythonModule));
-
-                var errors = ((AstScrapedPythonModule)mod).ParseErrors ?? Enumerable.Empty<string>();
-                foreach (var err in errors) {
-                    Console.WriteLine(err);
-                }
-                Assert.AreEqual(0, errors.Count(), "Parse errors occurred");
+                try {
+                    var fact = (AstPythonInterpreterFactory)analysis.Analyzer.InterpreterFactory;
+                    var interp = (AstPythonInterpreter)analysis.Analyzer.Interpreter;
+
+                    var mod = interp.ImportModule(interp.BuiltinModuleName);
+                    // TODO: CreateAnalysis specifies UseExistingCache = false,
```

Comment:
If printing this is important, I can save it and print it out in cleanup if test fails. If not important, I can delete the code.

Reply:
Yeah, we want this if the test fails.

Reply:
The `ImportModule` call above generates this file.